cesium_app/tests/conftest.py

```diff
@@ -78,16 +85,34 @@ def get(self, uri):
     return driver
 
 
+@pytest.fixture(scope='function', autouse=True)
+def reset_state(request):
+    def teardown():
+        models.DBSession().rollback()
+    request.addfinalizer(teardown)
+
+
 @pytest.fixture(scope='session', autouse=True)
-def remove_test_files(request):
+def delete_temporary_files(request):
     def teardown():
-        for f in models.File.select():
-            try:
-                os.remove(f.file.uri)
-            except:
-                pass
-    try:
-        models.db.connect()
-        request.addfinalizer(teardown)
-    except:  # skip teardown if DB not available
-        pass
+        shutil.rmtree(TMP_DIR, ignore_errors=True)
+    request.addfinalizer(teardown)
+
+
+register(ProjectFactory)
+register(DatasetFactory)
+register(DatasetFactory, "unlabeled_dataset")
+register(FeaturesetFactory)
+register(ModelFactory)
+register(PredictionFactory)
+register(PredictionFactory, "unlabeled_prediction")
+
+
+@pytest.fixture
+def unlabeled_dataset__name():
```

Comment:
This is the `pytest_factoryboy` syntax, I'll add some documentation for this. That's how you set the `.name` property of the fixture `unlabeled_dataset`.